src/routes/MapExplorer/MapExplorer.store.test.js

```diff
@@ -0,0 +1,20 @@
+import { fetchPosts } from './MapExplorer.store'
```

Comment:
Ah, I see you embraced snapshots here so disregard my earlier question, this answers it enough.